css/src/components/gradient.scss

```diff
@@ -88,3 +88,15 @@ $border-gradient-direction: (
 	-webkit-text-fill-color: transparent;
 	line-height: 1.4; // required because of background clip, avoid "g" and "y" getting cut off
 }
+
+.gradient-card {
+	border: $border-width-md solid $border !important;
```

Comment:
might be good to double check with Paige on this one, but wondering if the large border width might be preferable,  as the point of this is to be fairly obvious.

Reply:
Talked to Paige, decided to match the existing text gradient colors for consistency.